core/tests/protractor_utils/forms.js

```diff
@@ -624,15 +624,19 @@ var CodeMirrorChecker = function(elem) {
    *  - 'checked': true or false, whether the line has been checked
    */
   var _compareTextFromLine = function(
-      currentLineNumber, scrollTo, compareDict) {
+      currentLineNumber, scrollTo, Pane, compareDict) {
```

Comment:
I'm not sure about what's the `Pane` parameter here?
- Add proper name. (I can help you out if you can explain what exactly it's doing!)
- Use lowerCamel casing for variable names.
- Add details about the new parameter in the above docstring!

Reply:
Hi, Pane parameter selects the pane
where `first` and `last` can be passed to select a Pan
so if first is passed will select the first pane and scroll it 
its something like `lots_of_element.+pane+.scroll()` so this will be `lots_of_elements.first.scroll()`

Reply:
Refer to the above comments.

Reply:
Done